logeco_custom/__manifest__.py

```diff
@@ -10,8 +10,20 @@
     "category": "Uncategorized",
     "version": "12.0.1.0.0",
     "license": "AGPL-3",
-    "depends": ["housing_cooperative_base", "l10n_ch_states"],
-    "data": ["data/data.xml", "views/building.xml", "views/housing.xml"],
+    "depends": [
+        "housing_cooperative_base",
+        "l10n_ch_states",
+        "l10n_ch_payment_slip",
+        "l10n_ch_pain_credit_transfer",
+    ],
+    "data": [
+        "data/data.xml",
+        "views/building.xml",
+        "views/housing.xml",
+        "demo/demo.xml",  # fixme
```

Comment:
don't forget this

Reply:
It's on my wrapping up todo list, but tbh I don't know what to fix here. Does this refer to the 'noupdate'? I thought I should move this to the `demo/` directory, but that makes it crash?

Reply:
It should me moved in the __manifest__ from the data to the  demo section:

```python
{
    "name": "Housing Cooperative Custom",
...
    ],
    "data": [
        "data/data.xml",
        "views/building.xml",
        "views/housing.xml",
   
    ],
    "demo": [
        "demo/demo.xml",
    ],
...
}
```

Reply:
This gives an error for me. I'll look at it tomorrow..